### PR TITLE
Roman new dev (SettingsPage)

### PR DESCRIPTION
**Pull request by RomanWSU**
Created the settings page that has functional buttons.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -5,87 +5,69 @@
...
 
         <activity
             android:name=".closetPage.AllSavedOufitsPage"
             android:configChanges="orientation|keyboardHidden|screenSize"
-            android:exported="true">
-       
-       </activity>
-
-        <activity
-            android:name=".clothingFront"
```
Comment (k-odonnell) [inline app/src/main/AndroidManifest.xml:84]: make sure clothing front still has a section in manifest

File: app/src/main/java/com/example/mydigitalcloset/AboutUsPageActivity.java [hunk 2]
```diff
@@ -58,14 +58,14 @@ public void onClick(View view) {
               finish();
           }
       });
-        /*binding.settingsButton.setOnClickListener(new View.OnClickListener() {
```
Comment (k-odonnell) [inline app/src/main/java/com/example/mydigitalcloset/AboutUsPageActivity.java:61]: check functionality before merging

Comment (RomanWSU) [reply app/src/main/java/com/example/mydigitalcloset/AboutUsPageActivity.java:61]: Functionality does work and I am now able to merge

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (RomanWSU) [reply app/src/main/AndroidManifest.xml:84]: The clothing front still has a section in the manifest

**Pull request merged**